### PR TITLE
Cypress/FE-2147 validations date input

### PR DESCRIPTION
**Pull request by KatarzynaQA**
# Description
Added tests for classic and default date input validations stories

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Use scenario tags:
 5. `@ignore` and `@[bug-number]` should be removed after bug fix
 6. `@build` for pipeline - each build is verified by those tests
 7. `@accessibility` tests verifies accessibility violations
+8. `@validations` for validation component
```
Comment (DlgSHi) [inline cypress/README.md]: we should describe tags in this way -> `tests verify validation components`

File: cypress/features/regression/experimental/dateInputValidations.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    I want to change Date Input validations component properties
+
+    Background: Open Date Input validations component page
+        Given I open "Experimental Date Input" component page validations
+
+    @positive
+    @validations
+    Scenario Outline: Verify the <state> validation of Date Input component
+        When I type specific date "<date>"
+            And I click onto specific day "<specificDay>" via DayPicker for validation component
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInputValidations.feature]: to much spaces / tabs before `And` we should use VSC formatting

File: cypress/features/regression/experimental/dateInputValidations.feature [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            And I hover mouse onto icon
+        Then tooltipPreview on preview is set to "<text>"
+            And icon on preview is "<state>"
+
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInputValidations.feature]: Redundant empty line

File: cypress/features/regression/experimental/dateInputValidationsClassic.feature [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    I want to change Date Input validations classic component properties
+
+    Background: Open Date Input validations classic component page
+        Given I open "Experimental Date Input" component page validations classic
+
+    @positive
+    @validations
+    Scenario Outline: Verify the <state> validation of Date Input component
+        When I type specific date "<date>"
+            And I click onto specific day "<specificDay>" via DayPicker for validation component
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInputValidationsClassic.feature]: the same as above, spaces and empty lines between BDD code

File: cypress/features/regression/experimental/textboxValidations.feature [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    @validations
+    Scenario Outline: Verify the <state> validation of Textbox component
+        Given Type "<keyWord>" text into input
+            And I click outside of the component
+        When I hover mouse onto icon
+        Then tooltipPreview on preview is set to '<text>'
+            And icon on preview is "<state>"
+        Examples:
+            | state   | keyWord | text                                            |
+            | info    | info    | This value should be longer than 12 characters  |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/textboxValidations.feature]: why we should use the same values in `state` and `keyWord` columns?

File: cypress/features/regression/experimental/textboxValidationsClassic.feature [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    @validations
+    Scenario Outline: Verify the <state> validation of Textbox component
+        Given Type "<keyWord>" text into input
+            And I click outside of the component
+        When I hover mouse onto icon
+        Then tooltipPreview on preview is set to '<text>'
+            And icon on preview is "<state>"
+        Examples:
+            | state   | keyWord | text                                            |
+            | info    | info    | This value should be longer than 12 characters  |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/textboxValidationsClassic.feature]: the same here about same values

File: cypress/support/step-definitions/date-input-steps.js [hunk 7]
```diff
@@ -65,8 +65,16 @@ When('I choose date tomorrow via DayPicker', () => {
...
+});
+
 Then('the date is set to today', () => {
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'aria-label').should('contains', TODAY_CALENDAR);
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'class').should('contains', `${DAY_PICKER_PREFIX}today`);
   dayPickerDay(TODAY_CALENDAR).should('have.attr', 'class').should('contains', `${DAY_PICKER_PREFIX}selected`);
 });
+
+And('I click onto specific day {string} via DayPicker for validation component', (specificDay) => {
+  dayPickerDay(`${specificDay}`).click();
```
Comment (DlgSHi) [inline cypress/support/step-definitions/date-input-steps.js]: No need to user `${specificDay}` we can pass only the `specificDay` parameter by itselfs

Comment (KatarzynaQA) [reply cypress/support/step-definitions/date-input-steps.js]: fixed

File: cypress/features/regression/experimental/textboxValidationsClassic.feature (hunk 6 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/textboxValidationsClassic.feature]: fixed

File: cypress/features/regression/experimental/textboxValidations.feature (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/textboxValidations.feature]: fixed

File: cypress/features/regression/experimental/dateInputValidations.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateInputValidations.feature]: fixed

File: cypress/features/regression/experimental/dateInputValidations.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateInputValidations.feature]: fixed

File: cypress/README.md (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/README.md]: fixed

**Pull request closed without merging**